### DISCUSSION TITLE
StockAlertBot does not detect Pre-Order for Best Buy

### DISCUSSION THREAD

**Discussion by xmpierce1** [Prince25/StockAlertBot#118] [Requests]
This may be an enhancement but running the tool for pre-order for SwitchOLED and XBOX Halo with Best Buy. Below is an example of a live pre-order with an Amazon product to replicate.
https://www.bestbuy.com/site/blink-video-doorbell-wired-or-wire-free-two-way-audio-hd-video-and-alexa-enabled-black/6481230.p?skuId=6481230

**Comment by Prince25** [comment]
Just so I understand, do you want the bot to alert you when an item is ready to pre-order? How do you know if it'll be available for pre-order or just order?

**Comment by xmpierce1** [comment]
Yes. The Xbox series X halo edition is a good example

Sent from my mobile device


On Sep 30, 2021, at 5:44 PM, Prabhjot Singh ***@***.***> wrote:

﻿

Just so I understand, do you want the bot to alert you when an item is ready to pre-order? How do you know if it'll be available for pre-order or just order?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Prince25/StockAlertBot/discussions/118#discussioncomment-1411414>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AR3YQOGFSBLPRCKMCORLKVDUETK2DANCNFSM5FCJ7RWA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by Prince25** [comment]
Do preorders even have a limited quantity?

**Comment by xmpierce1** [comment]
Yep.. on hot items..


Microsoft Xbox Series X Halo Infinite Limited Edition Black C8Y-00023 - Best Buy<https://www.bestbuy.com/site/microsoft-xbox-series-x-halo-infinite-limited-edition-black/6477938.p?skuId=6477938>

**Reply by Prince25** [reply]
Okay, I'll add it to my TODOs.

**Comment by xmpierce1** [comment]
Thank you for fixing the Walmart site issue..
